README.md

```diff
@@ -53,6 +53,22 @@ To run a Rococo collator you will need to compile the following binary:
 cargo build --release -p polkadot-collator
 ```
 
+Otherwise you can compile it with 
+[Parity CI docker image](https://github.com/paritytech/scripts/tree/master/dockerfiles/ci-linux):
+
+```bash
+git clone https://github.com/paritytech/cumulus
```

Comment:
I think we can assume that the user cloned the repo and is in the proper folder. The previous commands (`cargo build`...) do assume that.
 
If we **really** want to explain how to clone the repo (I would suggest not to, as it adds maintenance if the repo moves or gets renamed) we could have a section at the beginning.